### PR TITLE
join is static sqw method

### PR DESCRIPTION
**Pull request by abuts**
This should be looked at after #1428 is merged. 

This is @oerc0122-style PR, containing only one change. `join` opeation for `sqw` object made static and various small changes related to search for join within the code base and fixing various small issues in the code (like typos) identified while looking for `join` within the code. 

I normally would not do such PR, because such changes are part of development, but because these changes produce working code at this stage, I do not mind making them into a separtate PR.

### CODE REVIEW

File: horace_core/sqw/@sqw/join.m
Comment (cmarooney-stfc) [inline horace_core/sqw/@sqw/join.m:1]: Appreciate that lines 2-3 are not part of your changes but they are maddeningly incomplete. Suggest adding at line 4:
" and should not be used to combine arbitrary sets of sqw objects; the wrong answer will be produced if this is done. For combining arbitrary sets of sqws, use combine_sqw (or appropriate name)"

Comment (abuts) [reply horace_core/sqw/@sqw/join.m:1]: this is old comment. I will modify it according to current situation

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]

File: horace_core/sqw/PixelData/@PixelDataBase/PixelDataBase.m [hunk 1]
```diff
@@ -139,8 +139,8 @@
         % memory or on disk and all additional properties describing
         % pix array, like its size, shape, alignment, etc
         metadata;
-        % the property contains or describes the pixel data array itself
-        % contains if the array fits area or describes it if the only
+        % the property contains or describes the pixel data array itself.
+        % Contains if the array fits area or describes it if the only
```
[the remark this answers is not in the document]
Comment (abuts) [reply horace_core/sqw/PixelData/@PixelDataBase/PixelDataBase.m:143]: somethins strange is happening with my commits -- they are not displayed on PR-s

[the remark this answers is not in the document]
Comment (abuts) [reply horace_core/sqw/PixelData/@PixelDataBase/PixelDataBase.m]: "contains" if array fits memory or "Describes" if ...

**Comment by abuts** [comment]
I hope I've clarified all unclear points and here we have very simple changes

**Pull request merged**